src/manager/identifiers.js

```diff
@@ -1,73 +1,113 @@
 import * as Ulid from '@kiosked/ulid'
 import * as emitter from '../utils/emitter'
-import { getCookie, getFromLs, getOrAddWithExpiration, setCookie } from '../utils/storage'
-import { isIframe, loadedDomain, parentHostname } from '../utils/page'
+import { loadedDomain } from '../utils/page'
 import { getLegacyId, getLegacyIdentifierKey } from '../utils/legacy'
 import { domainHash } from '../utils/hash'
+import { expiresInDays, strEqualsIgnoreCase } from '../utils/types'
+import { StorageStrategy } from '../model/storage-strategy'
 
-const NEXT_GEN_FP_NAME = '_lc2_duid'
+const NEXT_GEN_FP_NAME = '_lc2_fpi'
```

Comment:
is it ok, that we lose the cookies that are already set?

Reply:
Yes